coalib/output/ConsoleInteraction.py

```diff
@@ -599,13 +615,29 @@ def print_actions(console_printer, section, actions, failed_actions):
                             values for the action. If the user did
                             choose to do nothing, return (None, None).
     """
-    choice = choose_action(console_printer, actions)
-
+    if section.name == 'cli':
+        if section.get('reprompt').value == 'True' or
+        section.get('reprompt').value == 'False' or
+        section.get('reprompt').value == '':
+            if section.get('reprompt').value == 'True'
+            or section.get('reprompt').value == '':
```

Comment:
or see how does coala interpret boolean values when asking the user for a missing setting.